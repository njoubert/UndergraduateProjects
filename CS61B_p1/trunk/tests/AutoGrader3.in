Dmitri is the physicist.
Wolodja is not the physicist.
Wolodja lives in the ukrainian house.
There is a belorussian house.
The diplomat lives around here.
Who is the diplomat?
What do you know about Dmitri?

