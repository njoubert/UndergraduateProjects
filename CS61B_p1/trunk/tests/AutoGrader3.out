1. Dmitri is the physicist.
2. Wolodja is not the physicist.
3. Wolodja lives in the ukrainian house.
4. There is a belorussian house.
5. The diplomat lives around here.
Q: Who is the diplomat?
A: Wolodja is the diplomat.
Q: What do you know about Dmitri?
A: Dmitri is the physicist and lives in the belorussian house.
